src/data/majors/cs.ts

```diff
@@ -25,35 +25,38 @@ const csRequirements: readonly CollegeOrMajorRequirement[] = [
   },
   {
     name: 'Computer Science Core',
-    description: 'CS 2800 or CS 2802, CS 3110, CS 3410 or CS 3420, CS 4410, and CS 4820',
+    description:
+      'CS 2800 or CS 2802, CS 3110, CS 3410 or CS 3420, CS 3700 or CS 3780, CS 4410, and CS 4820',
     source: 'https://www.cs.cornell.edu/undergrad/csmajor',
     checker: includesWithSubRequirements(
       ['CS 2800', 'CS 2802'],
       ['CS 3110'],
       ['CS 3410', 'CS 3420'],
+      ['CS 3700', 'CS 3780'],
       ['CS 4820'],
       ['CS 4410', 'CS 4414']
     ),
     fulfilledBy: 'courses',
-    perSlotMinCount: [1, 1, 1, 1, 1],
+    perSlotMinCount: [1, 1, 1, 1, 1, 1],
     slotNames: [
       'CS 2800 or CS 2802',
       'CS 3110',
       'CS 3410 or CS 3420',
+      'CS 3700 or CS 3780',
       'CS 4820',
       'CS 4410 or CS 4414',
     ],
   },
   {
     name: 'CS Electives',
     description:
-      'Three 4000+ CS electives each at 3 credits. CS 4090, CS 4998, and CS 4999 are NOT allowed.',
+      'Two 4000+ CS electives each at 3 credits. CS 4090, CS 4998, and CS 4999 are NOT allowed. CS 3700 or CS 3780 allowed if not used in CS core.',
     source:
       'http://www.cs.cornell.edu/undergrad/rulesandproceduresengineering/choosingyourelectives',
     checker: [
       (course: Course): boolean => {
         if (
-          courseMatchesCodeOptions(course, ['CS 4090', 'CS 4998', 'CS 4999', 'CS 4410', 'CS 4820'])
+          courseMatchesCodeOptions(course, ['CS 4090', 'CS 4998', 'CS 4999', 'CS 4410', 'CS 4820', 'CS 3700', 'CS 3780'])
```

Comment:
See my previous comment. We don't want to add this, as it is possible that they could take CS 3700 and CS 3780, one for core and one for electives. There is a circumstance where a user could have one of these courses count towards their CS electives.